packages/components/src/components/transfer-amount-input/transfer-amount-input.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import Input from '../input';
+
+type TTransferAmountInput = {
+    className?: string;
+    currency: string;
+    disabled?: boolean;
+    initialValue?: number;
+    label?: React.ReactNode;
+};
+
+const TransferAmountInput = ({ className, currency, disabled = false, initialValue, label }: TTransferAmountInput) => {
+    const [value, setValue] = useState(0);
+
+    return (
+        <div className='transfer-amount-input-wrapper'>
+            <span>{label}</span>
+            <Input
```

Comment:
Not sure about how to add the trailing `"{currency}"` string for the input, like in Figma; the best option I've found so far is [this](https://stackoverflow.com/a/53060450)